### PR TITLE
ROX-22121: migrate mapping files to prod destination

### PR DESCRIPTION
**Pull request by daynewlee**
## Description

Use defintion.stackrox.io for mapping file _**only**_

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed
To be added

### CODE REVIEW

**Review by dcaravel — COMMENTED** [review]
With the move to the formal bucket, are there plans to change the file names/path in the contents of `mapping.zip` from `repomapping-tmp/*` to something seemingly less temporary?

File: central/scannerdefinitions/handler/handler.go [hunk 1]
```diff
@@ -212,7 +212,7 @@ func (h *httpHandler) getUpdater(key string) *requestedUpdater {
 		var urlStr string
 		switch key {
 		case mappingUpdaterKey:
-			urlStr, _ = url.JoinPath(v4StorageDomain, mappingFile)
+			urlStr, _ = url.JoinPath(scannerUpdateDomain, mappingFile)
```
[the remark this answers is not in the document]
Comment (daynewlee) [reply central/scannerdefinitions/handler/handler.go:215]: vulnerability bucket hasn't been migrated yet so we can probably keep it.

[the remark this answers is not in the document]
Comment (RTann) [reply central/scannerdefinitions/handler/handler.go:215]: I'm fine with keeping it for now, and we can delete once everything is migrated

File: .github/workflows/scanner-mapping-update.yaml [hunk 2]
```diff
@@ -3,40 +3,44 @@ name: Scanner mappings update
 on:
   schedule:
     - cron: "0 0 * * *"
+  push:
```
Comment (RTann) [inline .github/workflows/scanner-mapping-update.yaml:6]: From what I can tell, it's there now

```
curl -o mapping.zip https://definitions.stackrox.io/redhat-repository-mappings/mapping.zip
```

so just be sure to remove this prior to merge :)

**Review by RTann — APPROVED** [review]

**Pull request merged**